specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateLocations.json

```diff
@@ -16,7 +16,7 @@
     "application/json"
   ],
   "paths": {
-    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Update.Admin/updateLocations/": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Update.Admin/updateLocations/": {
```

Comment:
Path change is a breaking change. Can this be released in new API version?

Reply:
ARM paths should be case-insensitive, so this seems fine to me.

---
In reply to: [401298386](https://github.com/Azure/azure-rest-api-specs/pull/8899#discussion_r401298386) [](ancestors = 401298386)

Reply:
and also the guidance is to follow camelCasing